webroot/results/admin/persons_finish_unfinished_ACTION.php

```diff
@@ -125,6 +141,30 @@ function insertPerson( $newName, $newCountry, $newId ) {
   
   #--- Execute the command.
   dbCommand( $command );
+
+  if ( $oldCountry <> $newCountry || $oldName <> $newName){
+
+    #--- Guess countries iso2
+    $newCountryIso2 = getIso2FromCountryId( $newCountry );
+    if ( $newCountryIso2 ){
+      if ( $oldCountry == $newCountry ){
+        $oldCountryIso2 = $newCountryIso2;
+      } else {
+        $oldCountryIso2 = getIso2FromCountryId( $oldCountry );
+      }
+      if ( $oldCountryIso2 ){
+        #--- Build the command.
+        $command = "
+          UPDATE InboxPersons SET name='$newName', countryId='$newCountryIso2'
```

Comment:
Nevermind! I see that we've already created the row in Persons at this point, but later on, webroot/results/admin/scripts/import_persons.php is going to try to update the rows in Persons, and that script *does* treat InboxPersons.countryId as an iso.

My new question is this: Why does webroot/results/admin/scripts/import_persons.php silently fail to do anything when the updates it tries to do fail to actually update anything? Wouldn't we have caught this bug much sooner if that script failed in an obvious way when it cannot find a row to update in the Persons table?